src/Insect/Parser.purs

```diff
@@ -224,8 +224,8 @@ normalUnitDict = Dictionary
   , Q.week ==> ["weeks", "week"]
   , Q.month ==> ["months", "month"]
   , Q.year ==> ["years", "year"]
-  , Q.gram ==> ["grams", "gram", "g"]
-  , Q.meter ==> ["meters", "meter", "m"]
+  , Q.gram ==> ["grams", "gram", "grammes", "gramme", "g"]
```

Comment:
Thanks. It's only relevant for `gramme(s)` because `metre` is not a prefix of `meter`.